react/features/follow-me/middleware.js

```diff
@@ -104,11 +104,11 @@ function _onFollowMeCommand(attributes = {}, id, store) {
 
     // The Command(s) API will send us our own commands and we don't want
     // to act upon them.
-    if (participantSendingCommand.local) {
+    if (participantSendingCommand?.local) {
         return;
     }
 
-    if (participantSendingCommand.role !== 'moderator') {
+    if (participantSendingCommand && participantSendingCommand.role !== 'moderator') {
```

Comment:
Why this change? You should be able to use nullish coalescing now.

Reply:
because here we're interested in the case in which participant is not hidden, and we check the role. If particpantSendingCommand is undefined, it does not mean it's not a moderator.